.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Dear friend, we, the rest of the NativeScript community thank you for your
+contribution! Because we want to present a really nice, readable changelog with each
+release, please provide the following information:
+
+### Create a meaningful title
+Please, ensure your title is less than 50 characters and starts with a capital
+letter. We strive to follow the guidelines in the
+[How to Write a Git Commit Message] (http://chris.beams.io/posts/git-commit/)
+article for PR titles.
+
+### Does your commit message include the wording below to reference a specific issue in this repo?
+Fixes/Implements #[Issue Number].
+
+### Does your pull request have [unit tests](https://github.com/NativeScript/NativeScript/blob/master/running-tests.md)?
+If not, why?
```

Comment:
Instead of asking these questions, we can ask for steps to test the new fixes/features. How about - `Outline the steps to test or reproduce the changes in this Pull Request.